CHANGELOG.md

```diff
@@ -2,12 +2,19 @@
 
 ## dev
 
-Version <dev> of the agent adds [Roda](https://roda.jeremyevans.net/) instrumentation, introduces improved error tracking functionality by associating a transaction id with each error, and uses more reliable network timeout logic.
+Version <dev> of the agent adds [Roda](https://roda.jeremyevans.net/) instrumentation, adds a new `allow_all_headers` configuration parameter to permit capturing all HTTP headers, introduces improved error tracking functionality by associating a transaction id with each error, and uses more reliable network timeout logic.
 
 - **Feature: Add Roda instrumentation**
 
   [Roda](https://roda.jeremyevans.net/) is a now an instrumented framework. The agent currently supports Roda versions 3.19.0+. [PR#2144](https://github.com/newrelic/newrelic-ruby-agent/pull/2144)
 
+- **Feature: New allow_all_headers configuration parameter**
```

Comment:
[Our config doc](https://docs.newrelic.com/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/) calls them "options" rather than "parameters".

```suggestion
- **Feature: New allow_all_headers configuration option**
```

Reply:
Spoiler alert: There are a few other comments related to editing this changelog entry below. You don't need to address them all. The last one includes changes/questions referenced in the prior comments.